src/models/disassemblyoutput.cpp

```diff
@@ -303,27 +305,32 @@ DisassemblyOutput DisassemblyOutput::disassemble(const QString& objdump, const Q
     auto binary = findBinaryForSymbol(debugPaths, extraLibPaths, symbol);
     if (binary.isEmpty()) {
         disassemblyOutput.errorMessage +=
-            QApplication::translate("DisassemblyOutput", "Could not find %1").arg(symbol.binary);
+            QApplication::translate("DisassemblyOutput", "<qt>Could not find <tt>%1</tt>").arg(symbol.binary);
         return disassemblyOutput;
     }
     arguments.append(binary);
 
-    asmProcess.start(objdump, arguments);
+    asmProcess.start(processPath, arguments);
```

Comment:
ideally this here is also its own patch - whenever your commit message is something like "do A and B" then it's an indication that you should have split it up into two patches instead